src/vs/code/browser/workbench/workbench-dev.html

```diff
@@ -16,6 +16,15 @@
 		<!-- Workbench Icon/Manifest/CSS -->
 		<link rel="icon" href="/favicon.ico" type="image/x-icon" />
 		<link rel="manifest" href="/manifest.json">
+		<style>
```

Comment:
This needs to move into [`style.css`](https://github.com/Microsoft/vscode/blob/e1983faa78e623bbf7ec866d6cd467bf34f07bad/src/vs/workbench/browser/media/style.css#L43)

Reply:
👍